elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/metadata/models/TimeDimension.java

```diff
@@ -23,7 +23,7 @@
 @EqualsAndHashCode(callSuper = true)
 @Include(rootLevel = false, type = "timeDimension")
 @Value
-public class TimeDimension extends Dimension {
+public class TimeDimension extends Column {
```

Comment:
The javadoc for the class says TimeDimension is type of Dimension, should that be updated to reflect this change?